js/hview.js

```diff
@@ -624,6 +659,12 @@ $(function () {
         setupWindowTitle(currentAppInfo)
         setupBackButton(appInfo)
 
+        function moveChildNodes(newParent, oldParent) {
+            while (oldParent.childNodes.length > 0) {
```

Comment:
does this work?
```
newParent. replaceChildren(...oldParent.childNodes)
```

Reply:
Yes! That does work. I'll use that and delete the moveChildNodes function.